### PR TITLE
translated glossary/application-architect from en to fr

### PR DESCRIPTION
**Pull request by ryanzidago**

### CODE REVIEW

File: content/fr/docs/reference/glossary/application-architect.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: Application Architect
```
Comment (sftim) [inline content/fr/docs/reference/glossary/application-architect.md]: As this is a concept (and not really Kubernetes-specific or tied to code), I would be tempted to translate the title of this glossary entry. Anyone specifically searching the French localization for the English term will be able to find it even with the French spelling of architecte.

@ryanzidago what do you think?

Comment (ryanzidago) [reply content/fr/docs/reference/glossary/application-architect.md]: @sftim alright.
 
I translated the title section in French too.

File: content/fr/docs/reference/glossary/application-architect.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+aka:
+tags:
+- user-type
+---
+ Personne responsable de la conception haut niveau d'une application.
+
+<!--more-->
+
+Un architecte s'assure que l'implémentation d'une application lui permet d'interagir avec ses composants environnants, de manière maintenable et déployable à grande échelle. Les composants environnants comprennent les bases de données, les infrastructures de logging et autres micro-prestations.
```
Comment (remyleone) [inline content/fr/docs/reference/glossary/application-architect.md]: micro-services

**Pull request merged**